File: src/nl/rubensten/texifyidea/action/insert/InsertSubSubSectionAction.java
package nl.rubensten.texifyidea.action.insert;

import nl.rubensten.texifyidea.TexifyIcons;
import nl.rubensten.texifyidea.action.InsertEditorAction;

/**
 * @author Ruben Schellekens
 */
public class InsertSubSubSectionAction extends InsertEditorAction {

    public InsertSubSubSectionAction() {
        super("Subsubsection", TexifyIcons.DOT_SUBSUBSECTION, "\\subsubsection{", "}");
    }
}

File: src/nl/hannahsten/texifyidea/action/insert/InsertSubSubSectionAction.kt
package nl.hannahsten.texifyidea.action.insert;

import nl.hannahsten.texifyidea.TexifyIcons;
import nl.hannahsten.texifyidea.action.InsertEditorAction;

/**
 * @author Hannah Schellekens
 */
public class InsertSubSubSectionAction extends InsertEditorAction {

    public InsertSubSubSectionAction() {
        super("Subsubsection", TexifyIcons.DOT_SUBSUBSECTION, "\\subsubsection{", "}");
    }
}
